pkg/sql/pai_submitter.go

```diff
@@ -395,6 +396,55 @@ func (s *paiSubmitter) ExecuteEvaluate(cl *ir.EvaluateStmt) error {
 	return e
 }
 
+// TODO(sneaxiy): need to add some tests to this function, but it requires
+// optflow installed in docker image
+func (s *paiSubmitter) ExecuteOptimize(cl *ir.OptimizeStmt) error {
```

Comment:
`ExecuteOptimize` actually is not submitting jobs to PAI. Can put this function under `submitter.go`, call the function in here so that when `SQLFLOW_submitter` is configured to `pai`, we can also submit the job.

Reply:
Done.